pennylane/qchem/vibrational/christiansen_utils.py

```diff
@@ -143,7 +143,7 @@ def _local_cform_twomode_kinetic(rank, boscombos_on_rank, pes, all_mode_combos,
     """
     local_kin_cform_twobody = np.zeros(len(all_mode_combos) * len(boscombos_on_rank))
     for nn, (ii, jj) in enumerate(all_mode_combos):
-        # TODO: Skip unnecessary combinations
+        # Skip unnecessary combinations
```

Comment:
So was it only `if jj >= ii` that was considered unnecessary, or are there other unnecessary combinations that we don't have the context for?

Reply:
I asked @austingmhuang who wrote this, and he said that he just forgot to remove the TODO. Maybe he can confirm.